### PR TITLE
Blog - Jobs and Cron Jobs in Kubernetes

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac [Preview Link](https://645a56e1dff760313075e6d8--ephemeral-narwhal-3ac3b2.netlify.app/blog/kubernetes-jobs-cron-jobs/)

**Comment by msali123** [comment]
[Updated Link](https://645b7d08b058f8113176f19c--ephemeral-narwhal-3ac3b2.netlify.app/blog/kubernetes-jobs-cron-jobs/)

File: blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+- Jobs
+- Cron Jobs
+- Kubernetes
+- scheduled tasks
+- Management
+---
+
+Modern software applications require efficient and reliable management of jobs and scheduled tasks. As applications become more complex and Kubernetes clusters increase in size, automating repetitive tasks like backups, batch processing, and data analysis becomes essential.
+
+Kubernetes provides two critical resources to manage these tasks: Jobs and CronJobs. Jobs are ideal for running one-off or batch tasks, while a CronJob is perfect for scheduling recurring tasks. By using these resources, developers, DevOps engineers, and cluster administrators can focus on more critical tasks and ensure the reliability and efficiency of Kubernetes deployments.
```
Comment (balapriyac) [inline blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md:19]:
```suggestion
Kubernetes provides two critical resources to manage these tasks: **Jobs** and **CronJobs**. Jobs are ideal for running one-off or batch tasks, while a CronJob is perfect for scheduling recurring tasks. By using these resources, developers, DevOps engineers, and cluster administrators can focus on more critical tasks and ensure the reliability and efficiency of Kubernetes deployments.
```

File: blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+~~~
+
+The code snippet above does the following:
+
+- Creates a Kubernetes job named *hello-world* using the *batch/v1* API version.
+- Runs a container named *busybox* with the image *busybox*.
+- Executes the command *echo Hello World!* and then exits.
+
+<div class="notice--info">
+💡The *`restartPolicy: Never`* specifies that the container should not be restarted if it fails or exits.
```
Comment (balapriyac) [inline blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md:71]:
```suggestion
💡 The `restartPolicy: Never` specifies that the container should not be restarted if it fails or exits.
```

File: blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+Kubernetes provides two critical resources to manage these tasks: Jobs and CronJobs. Jobs are ideal for running one-off or batch tasks, while a CronJob is perfect for scheduling recurring tasks. By using these resources, developers, DevOps engineers, and cluster administrators can focus on more critical tasks and ensure the reliability and efficiency of Kubernetes deployments.
+
+Whether you're a seasoned developer or a Kubernetes novice, understanding Jobs and CronJobs is essential for efficient and reliable application management. In this tutorial, we'll explore how to run and schedule tasks with Kubernetes Jobs and CronJobs. You'll learn how to create (define the desired state), monitor (watch their completion status), and customize these powerful resources in Kubernetes!
+
+## Prerequisites
+
+To follow along in this tutorial, you'll need to have:
+
+- A Kubernetes cluster up and running (A single node cluster will work fine).
+- This tutorial uses a Linux machine with the Ubuntu 22.04LTS distro (Any other OS will work fine as long as you have Kubernetes set up).
```
Comment (balapriyac) [inline blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md:28]:
```suggestion
- This tutorial uses a Linux machine with the Ubuntu 22.04 LTS distro (Any other OS will work fine as long as you have [Kubernetes set up](/blog/k8s-dev-solutions/)).
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer.

**Comment by msali123** [comment]
Thanks Bala [Changes Done](https://6461e84ff630b321a5c388bd--ephemeral-narwhal-3ac3b2.netlify.app/blog/kubernetes-jobs-cron-jobs/)

File: blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+editor: Bala Priya C
+
+internal-links:
+- Jobs
+- Cron Jobs
+- Kubernetes
+- scheduled tasks
+- Management
+---
+
```
Comment (balapriyac) [inline blog/_posts/2023-06-14-kubernetes-jobs-cron-jobs.md:16]:
```suggestion

**We're [Earthly.dev](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article covers jobs and cron jobs in Kubernetes. If you want to know more about building in containers then [check us out](/).**
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
@msali123 added Earthly tie-in.

**Comment by msali123** [comment]
@balapriyac [Added Earthly tie-in](https://6463363139c2565b8418b856--ephemeral-narwhal-3ac3b2.netlify.app/blog/kubernetes-jobs-cron-jobs/)

**Comment by balapriyac** [comment]
@jalletto this is ready to merge.

**Pull request merged**